src/sizzle.js

```diff
@@ -559,10 +560,13 @@ support = Sizzle.support = {};
  * @returns {Boolean} True iff elem is a non-HTML XML node
```

Comment:
Maybe we could go ahead and fix this typo, iff you want.

Reply:
What typo? Do you think we should replace "[iff](https://en.wikipedia.org/wiki/iff)" with "if and only if"?

Reply:
I didn't know that's what it meant.

Reply:
BTW, while I've used this notation repeatedly I'd guess it's not common knowledge among programmers without theoretical background. Expanding that could increase intelligibility of the comment to the general public.